### ISSUE TITLE
I can confirm that the CyberChef versions after 8.8.1 are not loading after saved to the local drive 

### ISSUE THREAD

**Issue by wrmbr** [gchq/CyberChef#395]
### Summary
CyberChef versions after 8.8.1 version which is stable, will not load after being downloaded on the local drive. Browsers used Chrome 70 and Edge

Unable to set property 'cptable' of undefined or null reference
Filename: blob:null/b2e7a418-53a6-4ebf-9505-eac8f093f0a5
Line: 1828
Column: 785599
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36 Edge/17.17134
CyberChef version: 8.8.5s

**Comment by newpIDC** [comment]
Seeing the same behavior here

**Comment by Biemans** [comment]
Same here with Firefox 62.0:

```
TypeError: e is undefined
Filename: blob:null/018fe1d6-b1f4-4be4-b313-80f76cca07f2
Line: 1828
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:62.0) Gecko/20100101 Firefox/62.0
CyberChef version: 8.8.5s
```

**Comment by bwhitn** [comment]
This appears to be related to #388

**Comment by n1474335** [comment]
Fixed in #403. Thanks to @bwhitn.
